django3_saml2_nbplugin/backends.py

```diff
@@ -6,7 +6,7 @@
 # benefits of the REMOTE_AUTH_DEFAULT_GROUPS and
 # REMOTE_AUTH_DEFAULT_PERMISSIONS
 
-from utilities.auth_backends import RemoteUserBackend
+from netbox.authentication import RemoteUserBackend
```

Comment:
Maybe this could just be in a `try…except…` block like so:

```python
try:
    from netbox.authentication import RemoteUserBackend
except ImportError:
    from utilities.auth_backends import RemoteUserBackend
```